### DISCUSSION TITLE
Is it possible to perform MAPROT processing for CCP4?

### DISCUSSION THREAD

**Discussion by taka-aki-k** [project-gemmi/gemmi#209] [Q&A]
Hello.
I have a question.
For example, is it possible to use gemmi to do the following MAPROT process for CCP4?

`
maprot mapin [INPUT] wrkout  [OUTPUT] << eof
MODE FROM
CELL WORK
GRID WORK
XYZLIM
SYMM WORK 1
AVER
ROTAte matrix
Translate 0.0 0.0 0.0
eof
`

I looked at the official Gemmi documentation but couldn't figure it out.
I am new to structural biology. Can you just tell me what I can and can't do. I would appreciate it if someone could tell me what I need to know.

**Comment by wojdyr** [comment]
Gemmi can interpolate map data. But I don't what exactly that MAPROT call in the question does.

**Reply by taka-aki-k** [reply]
Thanks for the reply.
I'm afraid I may not be communicating it well.
I want to apply a virtual map of the size and grid width I specify to an already existing electronic density map file and cut out voxels at specified locations.
